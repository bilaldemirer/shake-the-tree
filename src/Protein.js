import React from "react";
import { Provider } from "react-redux";

// Components
import Tree from "./components/Tree";
import Basket from "./components/Basket";
import Apple from "./components/Apple";
import Button from "./components/Button";

import store from "./store";

const Protein = () => {
  return (
    <Provider store={store}>
      <Tree />
      <Basket />
      <Apple />
      <Button />
    </Provider>
  );
};

export default Protein;
