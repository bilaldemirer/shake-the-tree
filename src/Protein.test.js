import Protein from "./Protein";
import React from "react";
import { shallow } from "enzyme";
import { Provider } from "react-redux";

describe("Protein", () => {
  test("should match snapshot", () => {
    const wrapper = shallow(<Protein />);
    expect(wrapper.find(<Provider />));
    expect(wrapper).toMatchSnapshot();
  });
});
