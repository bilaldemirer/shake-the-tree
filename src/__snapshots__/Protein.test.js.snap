// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Protein should match snapshot 1`] = `ShallowWrapper {}`;
